### PR TITLE
Errors display if password and confirm password don't match when resetting password

### PR DESCRIPTION
**Pull request by Baonganbui123**

### CODE REVIEW

File: src/test/java/tests/ForgotPasswordTests.java [hunk 1]
```diff
@@ -40,4 +41,19 @@ public void tc012_ResetPasswordWhenPasswordResetTokenIsBlank() {
...
         Assert.fail();
     }
+
+    @Test(description = "Errors display if password and confirm password don't match when resetting password")
+    public void tc013_ResetPasswordWithPasswordAndConfirmPasswordDoNotMatching() {
+        LogHelpers.printInfo("Click on 'Login' tab");
+        loginPage.clickLoginTab();
+
+        LogHelpers.printInfo("Click on 'ForgotPassword' link");
+        forgotPasswordPage.clickForgotPasswordLink();
```
Comment (MinhQuan119) [inline src/test/java/tests/ForgotPasswordTests.java:51]: image

clickForgotPasswordLink method belongs to LoginPage class, not ForgotPasswordPage class

Comment (Baonganbui123) [reply src/test/java/tests/ForgotPasswordTests.java]: I changed it, please check again!

File: src/main/java/page_object/LoginPage.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
     private final By lblErrorPassword = By.cssSelector("div#content > p");
     private final By lblHeader = By.cssSelector("div#content > h1[align='center']");
 
+
```
Comment (MinhQuan119) [inline src/main/java/page_object/LoginPage.java:18]: redundant

Comment (Baonganbui123) [reply src/main/java/page_object/LoginPage.java:19]: Done, please check again!

**Review by MinhQuan119 — APPROVED** [review]

File: src/main/java/page_object/LoginPage.java [hunk 3]
```diff
@@ -44,6 +46,10 @@ private WebElement getLblHeader() {
         return DriverHelpers.getDriver().findElement(lblHeader);
     }
 
+    private WebElement getForgotPassword() {
```
Comment (tanmle) [inline src/main/java/page_object/LoginPage.java:49]: getLnk...

File: src/test/java/tests/ForgotPasswordTests.java [hunk 4]
```diff
@@ -32,7 +33,22 @@ public void tc012_ResetPasswordWhenPasswordResetTokenIsBlank() {
...
 
         LogHelpers.printInfo("Click on 'ForgotPassword' link");
-        forgotPasswordPage.clickForgotPasswordLink();
+        loginPage.clickForgotPasswordLink();
+
+        LogHelpers.printInfo("Enter the email address of the created account in Pre-condition");
+        forgotPasswordPage.resetPassword(newEmail);
+
+        LogHelpers.printInfo("Verify that error page displayed when click send instructions button");
+        Assert.fail();
```
Comment (tanmle) [inline src/test/java/tests/ForgotPasswordTests.java:42]: why don't you put a message for `Assert.fail`?

File: src/test/java/tests/ForgotPasswordTests.java [hunk 5]
```diff
@@ -32,7 +33,22 @@ public void tc012_ResetPasswordWhenPasswordResetTokenIsBlank() {
         loginPage.clickLoginTab();
 
         LogHelpers.printInfo("Click on 'ForgotPassword' link");
-        forgotPasswordPage.clickForgotPasswordLink();
+        loginPage.clickForgotPasswordLink();
+
+        LogHelpers.printInfo("Enter the email address of the created account in Pre-condition");
+        forgotPasswordPage.resetPassword(newEmail);
+
+        LogHelpers.printInfo("Verify that error page displayed when click send instructions button");
```
Comment (tanmle) [inline src/test/java/tests/ForgotPasswordTests.java:41]: We are not verifying the error page, the error page is unexpected.

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/main/java/page_object/LoginPage.java (hunk 3 above)
Comment (Baonganbui123) [reply src/main/java/page_object/LoginPage.java:49]: I fixed it, please check again.

File: src/test/java/tests/ForgotPasswordTests.java (hunk 4 above)
Comment (Baonganbui123) [reply src/test/java/tests/ForgotPasswordTests.java]: I putted a message for Assert.fail(), please check again!

File: src/test/java/tests/ForgotPasswordTests.java (hunk 5 above)
Comment (Baonganbui123) [reply src/test/java/tests/ForgotPasswordTests.java]: I fixed it, please check again!

**Review by tanmle — APPROVED** [review]

**Pull request merged**